t/01-driver.t

```diff
@@ -299,7 +299,11 @@ FIND: {
     is(scalar(@$elems),1, 'Got an arrayref of WebElements');
     my @array_elems = $driver->find_elements("//input[\@id='checky']");
     is(scalar(@array_elems),1, 'Got an array of WebElements');
-    is($elems->[0]->get_value(),$array_elems[0]->get_value(), 'and the elements returned are the same');
+    {
+        no warnings 'redefine';
+        local *Selenium::Remote::WebElement::get_value = sub { return "zippy"; };
```

Comment:
Not a fan of redefining the method you then immediately call.  Would be better to mock dependent methods in the method called.